bot/service/thorchain_network_service.py

```diff
@@ -29,18 +29,26 @@ def get_latest_block_height(node_ip=None) -> int:
     return int(get_node_status(node_ip)['result']['sync_info']['latest_block_height'])
 
 
+def is_block_height_stuck(main_node_ip, reference_node_ip) -> bool:
+    main_block_height = get_latest_block_height(main_node_ip)
+    reference_block_height = get_latest_block_height(reference_node_ip)
+
+    # block heights within plus minus 1 block are considered equal (network latency could lead to one block difference)
+    return not (main_block_height == reference_block_height or main_block_height == reference_block_height + 1 or main_block_height == reference_block_height - 1)
```

Comment:
Readability could be improved by this suggestion:

```suggestion
    diff_block_height = abs(main_block_height - reference_block_height)
    return diff_block_height > 1
```

Reply:
Maybe also explain briefly why the reference_node_ip is not checked for is_block_height_stuck as a comment. Otherwise, we will be misinterpreting this logic when we look at it again in 2 months 👀